torch/utils/benchmark/utils/timer.py

```diff
@@ -142,13 +243,34 @@ def stop_hook(times) -> bool:
             task_spec=self._task_spec
         )
 
-    def blocked_autorange(self, callback=None, min_run_time=0.2):
+    def blocked_autorange(
+        self,
+        callback: Optional[Callable[[int, float], NoReturn]] = None,
+        min_run_time: float = 0.2,
+    ) -> common.Measurement:
+        """Measure many replicates while keeping timer overhead to a minimum.
+
+        This method repeats `stmt` until a time quota (set by `min_run_time`)
+        has been met. It dynamically chooses the block size by balancing two
```

Comment:
block size is a new concept that has not been defined

Reply:
Done.